### PR TITLE
Mark round the clock as departed after 12 hours

### PR DESCRIPTION
**Pull request by reynders**

### CODE REVIEW

File: service/src/main/kotlin/fi/espoo/evaka/attendance/StaffAttendanceQueries.kt [hunk 1]
```diff
@@ -522,6 +522,34 @@ WHERE a.departed IS NULL AND a.arrived < :startOfDay AND a.group_id = g.id AND N
...
+UPDATE staff_attendance_external a
+SET departed = a.arrived + interval '24 hours'
+FROM daycare_group g
+JOIN daycare d ON g.daycare_id = d.id
+WHERE a.departed IS NULL AND a.arrived + INTERVAL '24 hours' <= :now AND a.group_id = g.id AND d.round_the_clock
+        """
+                .trimIndent()
+        )
+        .bind("now", now)
+        .execute()
```
Comment (akheron) [inline service/src/main/kotlin/fi/espoo/evaka/attendance/StaffAttendanceQueries.kt:552]: Everything this function does should be documented in the wiki

**Review by akheron — APPROVED** [review]

**Review by akheron — CHANGES_REQUESTED** [review]
Discussed in daily: This cannot be merged as-is, the logic will be revised in planning

File: service/src/main/kotlin/fi/espoo/evaka/attendance/StaffAttendanceQueries.kt (hunk 1 above)
Comment (reynders) [reply service/src/main/kotlin/fi/espoo/evaka/attendance/StaffAttendanceQueries.kt:558]: I added a wiki entry (to be merged after this PR)

File: service/src/main/kotlin/fi/espoo/evaka/attendance/StaffAttendanceQueries.kt [hunk 2]
```diff
@@ -471,6 +472,7 @@ fun Database.Read.getGroupsForEmployees(
         .bind("employeeIds", employeeIds)
         .toMap { columnPair("employee_id", "group_ids") }
 
+// TODO
```
Comment (akheron) [inline service/src/main/kotlin/fi/espoo/evaka/attendance/StaffAttendanceQueries.kt:475]: TODO?

Comment (akheron) [inline service/src/main/kotlin/fi/espoo/evaka/attendance/StaffAttendanceQueries.kt:475]: TODO?

**Review by akheron — APPROVED** [review]
One stray comment, otherwise LGTM

**Pull request merged**